### ISSUE TITLE
New widget to plot the training loss

### ISSUE THREAD

**Issue by salvacarrion** [biolab/orange3-recommendation#3]
Usually is a good practice to analyze the evolution of the loss to understand how the model is being optimised. To achieve this, we can either output the loss as an array when the training finish or send it to the chart in a online way, having the possibility to stop the training at any given moment.
![main-qimg-d23fbbc85b7d18b4e07b7942ecdfd856](https://cloud.githubusercontent.com/assets/6052046/17718476/70669dee-6414-11e6-9e80-7b70aa595fb4.png)
